.github/workflows/release.yml

```
@@ -1,14 +1,22 @@
 name: Production Release
 
 on:
-  release:
-    types: [published]
+  workflow_dispatch:
+    branches:
```

actionlint (1.7.7)

expected "inputs" key for "workflow_dispatch" section but got "branches"

(syntax-check)